pcdsdevices/mpod_apalis.py

```diff
@@ -246,6 +297,54 @@ def set_current_ramp_speed(self, ramp_speed):
         """
         self.current_ramp_speed.put(ramp_speed)
 
+    @model.sub_value
+    def _stash_model(self, value, **kwargs) -> None:
+        self._model_str = self.model.get()
+
+    @property
+    def polarity(self) -> Polarity:
+        if self._model_str.endswith("x"):
+            return Polarity.BIPOLAR
+        elif self._model_str.endswith("p"):
+            return Polarity.POSITIVE
+        elif self._model_str.endswith("n"):
+            return Polarity.NEGATIVE
+
+        raise ValueError(f"Unable to determine module polarity: {self._model_str}")
```

Comment:
Even without the spam bug, we might be at risk to see this at startup depending on the callback order, since the default value is not valid.

Reply:
Ah a fair point.  I wanted to preserve this check but maybe it's not worth the complexity.

Reply:
removed this polarity logic entirely